WooCommerce/Classes/ViewRelated/Products/Edit Product/EditableProductVariationModel.swift

```diff
@@ -178,7 +178,7 @@ extension EditableProductVariationModel: ProductFormDataModel, TaxClassRequestab
     }
 
     func isImageDeletionEnabled() -> Bool {
```

Comment:
I'm just curious. Since this is always going to return true, would it make sense to declare it just as a let constant?

Reply:
Good question. One of the reasons is that in `EditableProductModel` and `EditableProductVariationModel` we are using the properties for returning the properties related to a product or a variation, and methods for managing the visibility logics. In reality, we can transform everything into a property in these classes, but maybe using properties and methods makes the roles more clear.